cmd/init.sh

```diff
@@ -41,6 +41,10 @@ sed -i 's/Test-Cluster-TiFlash-Summary/'$TIDB_CLUSTER_NAME'-TiFlash-Summary/g'
 cp /tmp/tiflash_proxy_summary.json $GF_PROVISIONING_PATH/dashboards
 sed -i 's/Test-Cluster-TiFlash-Proxy-Summary/'$TIDB_CLUSTER_NAME'-TiFlash-Proxy-Summary/g' $GF_PROVISIONING_PATH/dashboards/tiflash_proxy_summary.json
 
+# TiCDC dashboard
+cp /tmp/ticdc.json $GF_PROVISIONING_PATH/dashboards
+sed -i 's/Test-Cluster-TiCDC/'$TIDB_CLUSTER_NAME'-TiCDC/g' $GF_PROVISIONING_PATH/dashboards/ticdc.json
+
 # To support monitoring multiple clusters with one TidbMonitor, change the job label to component
 sed -i 's%job=\\\"tiflash\\\"%component=\\"tiflash\\"%g' $GF_PROVISIONING_PATH/dashboards/*.json
 sed -i 's%job=\\\"tikv-importer\\\"%component=\\"importer\\"%g' $GF_PROVISIONING_PATH/dashboards/*.json
```

Comment:
We need to replace the `job=` strings in the cdc.json.

Reply:
Should `by (job, instance)` here https://github.com/pingcap/ticdc/blob/master/metrics/grafana/ticdc.json#L80 also be updated?

Reply:
I have added sed command to update expression like: `by (...,job,...)`. I also added a sed command to update legend format, such as `"legendFormat": "{{job}}"` in grafana dashboard definition.